### PR TITLE
Feature/acq 971 new stateofcs

### PR DESCRIPTION
**Pull request by dmantonyuk**
Create new advocacy.code.org/stateofcs page

## Links

https://codedotorg.atlassian.net/browse/ACQ-971

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
State of cs

File: pegasus/sites.v3/advocacy.code.org/public/stateofcs-temp.haml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      %img{src: "/images/fit-400/ECEP_Logo.jpg", style: "width: 200px; padding: 10px", alt: "ECEP Logo"}
+  .clear
+
+.website__nav{style: "margin-bottom: 20px;"}
+  .website__nav__links
+    %a{href: "/stateofcs"}
+      Home
+    %a{href: "/stateofcs/data"}
+      Data 
+    %a{href: "/stateofcs/thanks"}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/advocacy.code.org/public/stateofcs-temp.haml]: This link need to point to `/stateofcs/thankyou`

**Review by kelbyhawn — COMMENTED** [review]
Overall this looks good! The filename needs to be updated so it works as the live page, and I'm not sure where we're at with the map.

**Comment by dmantonyuk** [comment]
2023 State of CS Report - CS Advocacy

**Review by kelbyhawn — APPROVED** [review]
Looks great! 🎉

**Pull request merged**